recipes/mlpack/meta.yaml

```diff
@@ -0,0 +1,61 @@
+{% set version = "3.0.2" %}
+
+
+package:
+    name: mlpack
+    version: {{ version }}
+
+source:
+    fn: mlpack-{{ version }}.tar.gz
+    url: http://www.mlpack.org/files/mlpack-{{ version }}.tar.gz
+    sha256: 69e81178839594049d84526b8f9612825f238f7cfe1f6d579bd6461ed48e5aa0
+
+build:
+  number: 0
+  skip: true  # [win]
+
+requirements:
+  build:
+    - {{ compiler('c') }}
+    - {{ compiler('cxx') }}
+    - cmake
+    - setuptools
+    - cython
+  host:
+    - boost
+    - python
+    - numpy
+    - pandas
+    - armadillo
```

Comment:
i'd move `cython` and `pip` to this section.  cython is tied to your python version

Reply:
Got it!